### PR TITLE
Add peruvian channels

### PR DESCRIPTION
**Pull request by sguinetti**
This pull request adds several channels from Peru. They were initially proposed to be added to TDTChannels but some channels were rejected as out of scope.

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -11,11 +11,29 @@ http://cdnh4.iblups.com/hls/OVJNKV4pSr.m3u8
 https://cdnhd.iblups.com/hls/ptv2.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/B89KuHO.png" group-title="",Pax TV (480p)
 https://d2fxrfbiedz1tm.cloudfront.net/livepaxtv/smil:PC.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/uncanalinteligente/picture?width=320&height=320" group-title="",UCI
+http://cdn.win.pe:8080/UCI/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline channels/pe.m3u:15]: Link doesn't work.
Captura de pantalla 2021-02-20 a las 14 38 58

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u]: Updated and fixed: http://170.79.102.223/UCI/index.m3u8

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@sguinetti Great job!

**Pull request merged**